src/components/common/ActivityButtons.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <div class="flex flex-col flex-nowrap w-full">
+    <RadioButtons
+      :icons="icons"
+      @change="$emit('update:modelValue', value)"
+      v-model="value"
+    />
+  </div>
+</template>
+<script>
+import RadioButtons from "@/components/common/RadioButtons";
+import ShoeIcon from "@/components/common/icons/shoe.vue";
+import RollerbladesIcon from "@/components/common/icons/rollerblades.vue";
+import BicycleIcon from "@/components/common/icons/bicycle.vue";
+import { ref } from "vue";
+
+export default {
+  name: "ActivityButtons",
+  components: {
+    RadioButtons,
+  },
+  setup() {
+    const value = ref({
+      name: "",
+      kcal: null,
+    });
+    const icons = {
```

Comment:
nie mieszajmy danych z logiką aplikacji. Rozumiem, że normalnie coś takiego by z backnedu przychodziło, ale skoro nie, to sądzę, że powinieneś tego jsona wydzielić do osobnego pliku (może nawet jakiś katalog na dane) i tylko zaimportować tutaj